app/modbus.mjs

```diff
@@ -29,6 +29,28 @@ const AVAILABLE_SETTINGS = {
     'temperatureTarget': 135,
 }
 
+export let ALARMS = {
```

Comment:
Maybe this should be `AVAILABLE_ALARMS` to match the other constants here?

Reply:
true